filament/src/details/Engine.cpp

```diff
@@ -835,8 +835,10 @@ FMaterialInstance* FEngine::createMaterialInstance(const FMaterial* material,
     return p;
 }
 
-FMaterialInstance* FEngine::createMaterialInstance(const FMaterial* material) noexcept {
-    FMaterialInstance* p = mHeapAllocator.make<FMaterialInstance>(*this, material);
+FMaterialInstance* FEngine::createMaterialInstance(const FMaterial* material,
+                                                   const char* name) noexcept {
+  FMaterialInstance* p =
```

Comment:
nit: 4 spaces here

Reply:
Fixed spacing as suggested. Thanks for pointing that out!